src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { Section } from '@site/src/pages/_components/section';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import {
+  Switcher,
+  SwitcherButton,
+  SwitcherContent,
+  SwitcherList,
+} from '../what-is-wasmcloud/switcher';
+import { SectionSubheading } from '@site/src/pages/_components/section-subheading';
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable Wasm components and run
+      them—resiliently and efficiently—across any cloud, Kubernetes, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Applications Faster',
```

Comment:
```suggestion
        title: 'Faster Development Cycles',
```
For someone skimming, headings that depend on the larger header ("Build...Applications Faster") might be confusing because they don't standalone. Suggesting phrasing that works either way.

Reply:
Yep, agreed. The button has no connection to the titles apart from theme. No one is going to read it as a single senctence.

Reply:
I want to try and capture the "pure functional business logic" aspect here but I'm struggling to find a few words for that. E.g. when you build with reusable application components you just implement business logic, not boilerplate